docs/user-guide/walkthrough/README.md

```diff
@@ -134,7 +134,7 @@ For this example, we'll be creating a Redis pod, with a named volume and volume
       emptyDir: {}
   ```
 
-1. Define a volume mount within a container definition:
+2. Define a volume mount within a container definition:
```

Comment:
not sure I understand @lavalamp but successive 1.'s in a list is markdown for "make this a numbered list and increment each number". It's fine to be explicit as well though but this change has no effect on the processed doc.